packages/flutter/lib/src/material/flat_button.dart

```diff
@@ -108,6 +108,10 @@ import 'theme_data.dart';
 ///  * [RawMaterialButton], the widget this widget is based on.
 ///  * <https://material.io/design/components/buttons.html>
 ///  * Cookbook: [Build a form with validation](https://flutter.dev/docs/cookbook/forms/validation)
+@Deprecated(
+  'Use TextButton instead. See the migration guide in flutter.dev/go/material-button-migration-guide). '
+  'This feature was deprecated after 1.25.0-8.1.pre'
```

Comment:
```suggestion
  'This feature was deprecated after v1.25.0-8.1.pre.'
```
Here and below, I think this is the format the linter is looking for.

Reply:
Thanks, hopefully I've finally got it.